docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -152,39 +153,45 @@ To illustrate some of these improvements, the following table display a benchmar
+Note that these results are based on raw filtering speed and not end-to-end operation (which includes user typing, filtering, and rerender).
 :::
 
 ## What's next?
 
 We're now working on the next MUI X major, v7, which will include many new features and improvements.
-
 Most notably:
 
-- DateTime and TimeRangePicker
-- Improved accessibility on date and time fields
-- New UI for column management for the [`DataGrid`](/x/react-data-grid/)
-- Pivoting for the [`DataGridPremium`](/x/react-data-grid/#premium-plan)
+- [Date and Time pickers](/x/react-date-pickers/) iteration:
+  - New component: `DateTime` and `TimeRangePicker`
+  - Improve accessibility on date and time fields
+- [Data Grid](/x/react-data-grid/) iterations:
+  - New UI for column management
+  - Pivoting for the [Premium](/x/react-data-grid/#premium-plan) version
 
-For the new components, we will keep expanding our portfolio with new chart types such as [Heat map](/x/react-charts/heat-map/), [Funnel](/x/react-charts/funnel/), [Gantt](/x/react-charts/gantt/), and explore virtualization and other advanced use cases for Tree view.
+We will also keep expanding our portfolio of charts, with types such as [Heat map](/x/react-charts/heat-map/), [Funnel](/x/react-charts/funnel/), and [Gantt](/x/react-charts/gantt/), and explore virtualization and other advanced use cases for the Tree view component.
 
-We encourage you to upvote issues on GitHub to help us prioritize. Your input directly influences our development schedule, so don't hesitate to let us know what matters most to you!
-You can use the following list for quick access to each components issues.
+We encourage you to upvote issues on GitHub to help us prioritize.
+Your input directly influences our development schedule, so don't hesitate to let us know what matters most to you!
+Use the following list to quickly access each component's issues:
 
 - [Charts](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+charts%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Tree View](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+treeview%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Data Grid](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+data+grid%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Date and Time pickers](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+pickers%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 
-The next major release (v7) will be available through alpha and beta versions until March 2024, when it's planned for it to be officially promoted to the new stable version. At the same time, v6 will transition to [Long-Term Support](/versions/#long-term-support-lts), while support for v5 will be discontinued.
+The next MUI X major version, v7, is available through alpha and beta versions until March 2024, when we plan to promote it to stable.
```

Comment:
I was also wondering here whether the v7 is _already_ available or if it _will_ be some time? 
My initial understanding was the former, hence my edit — but correct me if I'm wrong!